docs/development/updating-dependencies.md

```diff
@@ -7,6 +7,7 @@ Especially during the alpha and beta phases, we expect you to keep all Tauri dep
 ## Automatic updates
 
 The Tauri JS CLI has a command to install and update all needed dependencies, just run `tauri deps install` or `tauri deps update`.
+**Note**: This command is unavailable in CLI versions starting from `v1.0.0-rc.0`
```

Comment:
Probably remove the section altogether. I don't see us providing that command again.

Reply:
That was meant for users that want to upgrade from beta.8

Thinking about it it's probably better if they do it manually too 👍